hive/src/main/java/com/netease/arctic/hive/op/OverwriteHiveFiles.java

```diff
@@ -191,20 +204,31 @@ public void commit() {
     }
   }
 
-  private void setHiveLocations() {
+  private void setPartitionProperties() {
```

Comment:
How about rename `setPartitionProperties` to `commitPartitionProperties`?

Reply:
right, i woud take it